doc/release-notes-1.3.0.rst

```diff
@@ -0,0 +1,593 @@
+* Added nRF Oberon as a backend in the :ref:`nrfxlib:nrf_security`.
+
+NFC
+---
+
+* Added a :ref:`nfc_ndef_le_oob_rec_parser_readme` for decoding data used for Bluetooth LE OOB pairing.
+* Added support for nRF5340 in the :ref:`nrf-nfc-system-off-sample` sample.
+* Aligned file and API naming in the :ref:`lib_nfc_ndef` libraries.
+
+Immutable bootloader
+--------------------
+
+* Exposed :cpp:func:`fw_info_ext_api_provide` as an :ref:`external API <doc_fw_info_ext_api>`, so that :doc:`mcuboot:index` can use it to provide external APIs from the :ref:`bootloader` to its images.
+  This means that requesting external APIs in applications works even if MCUboot is included.
+* Fixed a bug so that the :ref:`bootloader` works with nRF5340 SPU flash regions.
+* Added a :ref:`doc_bl_storage` library:
+
+  * Renamed :file:`provision.h` and :file:`provision_flash.h` to ``bl_storage`` and allowed including the library in the application.
+  * Added documentation and tests.
+  * Added a monotonic version counter.
+    The immutable bootloader will not boot an application that has a lower version than the monotonic counter.
+
+Secure Partition Manager (SPM)
+------------------------------
+
+* Added support for disabling some services in the :ref:`secure_services` sample.
+  It now works in more bootloader configurations.
+* Disabled :option:`CONFIG_SPM_SERVICE_PREVALIDATE` in the :ref:`lib_spm` library, because this option requires the immutable bootloader.
+* Updated the :ref:`lib_spm` library to make it compatible with nRF5340 (with or without `anomaly 19`_). (?)
+
+CPU load measurement
+--------------------
+
+Added :ref:`cpu_load`, a debug module for measuring CPU load.
+
+iCalendar parser
+----------------
+
+Added :ref:`icalendar_parser_readme`, a library to parse iCalendar data format.
+
+MCUboot
+=======
+
+* Added external flash secondary slot MCUboot.
+  See :ref:`ug_bootloader_flash`.
+* Added an option to build Ed25519 signature validation without using mbedTLS, by relying on a bundled tinycrypt-based SHA-512 implementation.
+* Replaced CBOR decoding in serial recovery with code generated from a CDDL description.
+  This results in a flash footprint reduction of more than 1 KB while serial recovery is enabled.
+* Added support for X25519-encrypted images.
+* Added rollback protection.
+  There is support for a hardware rollback counter that must be provided as part of the platform, as well as a software solution that protects against some types of rollback.
+* Added an optional boot record in shared memory to communicate boot attributes to code that is run later.
+* Added a cleanup of the Arm core before the application is booted.
+* Allowed recovery over USB CDC ACM with logs enabled.
+* Various fixes to work with the latest Zephyr version.
+
+Build system
+============
+
+* Added support for external flash in the :ref:`secure_partition_manager` sample. (? or is this a duplicate of the last bullet?)
```

Comment:
@carlescufi did you add the last bullet point on purpose, or is it a duplicate?

Reply:
This is a duplicate, please remove

Reply:
Removing Secure Partition Manager and leaving Partition Manager in.